api_tests/tests/dry/lightning_routes.ts

```diff
@@ -1,13 +1,16 @@
-// These are stateless tests -- they don't require any state of the cnd and they don't change it
-// They are mostly about checking invalid request responses
-import "chai/register-should";
-import "../lib/setup_chai";
-import { oneActorTest } from "../lib/actor_test";
+/**
+ * @config { "logDir": "lightning_routes" }
+ */
+
 import { expect } from "chai";
+import { oneActorTest } from "../../lib/actor_test_new";
 
-setTimeout(async function() {
-    describe("Lightning routes", () => {
-        oneActorTest(
+// ******************************************** //
+// Lightning routes                               //
+// ******************************************** //
+describe("Dry - lightning routs", () => {
+    it("lightning-routes-post-eth-lnbtc-return-400", async function() {
+        await oneActorTest(
             "lightning-routes-post-eth-lnbtc-return-400",
```

Comment:
It would be nice to remove this duplication. IIRC, the spike contains a way of obtaining the current test name if it is run by jest. You could use that inside `nActorTest` to avoid having to pass the same string again.

Reply:
Done.